tests/fixtures/jars/expected-fixtures-jars.json

```diff
@@ -0,0 +1,23 @@
+{
+  "plugin": { "name": "bundled:maven", "runtime": "unknown" },
+  "package": {
+    "name": "fixtures.jars:spring-core-5.1.8.RELEASE.jar",
```

Comment:
Can you rename this file to match the jar (or rename the jar to match the file) so it's easier to see what expected goes with what fixture?

Reply:
yep will do